### PR TITLE
chore: fix snowflake license

### PR DESCRIPTION
**Pull request by simon824**
subtask of #2095

### CODE REVIEW

**Comment by imbajin** [comment]
Thanks, I'll check it with doc again

**Review by javeme — APPROVED** [review]

File: hugegraph-core/src/main/java/org/apache/hugegraph/backend/id/SnowflakeIdGenerator.java [hunk 1]
```diff
@@ -1,18 +1,14 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with this
- * work for additional information regarding copyright ownership. The ASF
- * licenses this file to You under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance with the License.
+ * Copyright 2010-2012 Twitter, Inc.
```
Comment (javeme) [inline hugegraph-core/src/main/java/org/apache/hugegraph/backend/id/SnowflakeIdGenerator.java:2]: according to the mentor's tip, we can keep two or more licenses in a file.

**Review by imbajin — CHANGES_REQUESTED** [review]
@zyxxoo we should also check other license header problem in the repo 

keep this PR open & update it until all header problems done

File: LICENSE [hunk 2]
```diff
@@ -200,3 +200,11 @@
...
    limitations under the License.
 
+========================================================================
+Apache 2.0 licenses
+========================================================================
+
+The following components are provided under the Apache License. See project link for details.
+The text of each license is the standard Apache 2.0 license.
+
+hugegraph-core/src/main/java/org/apache/hugegraph/backend/id/SnowflakeIdGenerator.java files from https://github.com/twitter-archive/snowflake
```
Comment (zyxxoo) [inline LICENSE:210]: remove "files"

Comment (zyxxoo) [reply LICENSE:210]: we need add this license to release-docs/licenses and release-doc/notices

**Review by javeme — DISMISSED** [review]

**Comment by javeme** [comment]
please check if we need address: https://github.com/apache/incubator-hugegraph/pull/2093#discussion_r1095593557

**Review by zyxxoo — DISMISSED** [review]

File: LICENSE (hunk 2 above)
Comment (imbajin) [reply LICENSE]: > we need also add this license to `release-docs/LICENSE` and `release-doc/NOTICES`

no `NOTICE` in the original project, so only refer it in LICENSE

**Review by imbajin — APPROVED** [review]

**Review by corgiboygsj — APPROVED** [review]

**Pull request merged**